src/repositories/donationRepository.ts

```diff
@@ -387,12 +387,20 @@ export const getPendingDonationsIds = (): Promise<{ id: number }[]> => {
       hours: Number(process.env.DONATION_VERIFICAITON_EXPIRATION_HOURS),
     })
     .toDate();
+
   return Donation.find({
-    where: {
-      status: DONATION_STATUS.PENDING,
-      isFiat: false,
-      createdAt: MoreThan(date),
-    },
+    where: [
+      {
+        status: DONATION_STATUS.PENDING,
+        isFiat: false,
+        createdAt: MoreThan(date),
+      },
+      {
+        status: DONATION_STATUS.PENDING,
+        isFiat: false,
+        importDate: MoreThan(date),
```

Comment:
Is this WHERE correct? Its the same expression twice. Maybe one of them is LessThan?

Reply:
Nvm its ok Thanks for clarifying in chat